crates/pallet-balanced-currency-swap-bridges-initializer/src/lib.rs

```diff
@@ -39,9 +39,6 @@ type EvmBalanceOf<T> =
 /// The current storage version.
 const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
 
-/// The current bridges initializer version.
-pub const CURRENT_BRIDGES_INITIALIZER_VERSION: u16 = 1;
```

Comment:
I think we should still keep an internal version though. It just wouldn't be bumped this time, but the reason why we introduced it in the first place - to keep track of which algorithm we used to balance out the bridges - it still very valid.

Reply:
Returned.